src/lib/modules/Items/backoffice/ItemListEntry.js

```diff
@@ -6,23 +6,29 @@ import _isEmpty from 'lodash/isEmpty';
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
-import { Grid, Header, Item, List } from 'semantic-ui-react';
+import { Grid, Header, Item, List, Button } from 'semantic-ui-react';
 
 class ItemCirculation extends Component {
   render() {
-    const { circulation } = this.props;
+    const { circulation, showPreviousLoan } = this.props;
```

Comment:
this is a fantastic idea to change the prefix description!
the prop I think in this case is good enough, the next time think about something more extendable, 
like `loanLinkPrefix` `loanReferencePrefix`, then it will not be connected only with the state of the loan, but you can pass whatever description you need as a prop, and it becomes more flexible